cpu/atmega_common/Makefile.dep

```diff
@@ -0,0 +1,24 @@
+# expand atmega_pcint module
+ifneq (,$(filter atmega_pcint,$(USEMODULE)))
+    USEMODULE += atmega_pcint0 atmega_pcint1 atmega_pcint2
```

Comment:
With only `atmega_pcint0` and `atmega_pcint1` now being supported on all AVRs, this needs to be adapted. Also: The indent here needs to be two spaces, instead of four. (The Makefiles have a different code style than the C files.)

How about this:

```C
ifneq (,$(filter atmega_pcint,$(USEMODULE)))
  USEMODULE += atmega_pcint0 atmega_pcint1
  ifneq (,$(filter atmega_pcint2,$(FEATURES_PROVIDED)))
    USEMODULE += atmega_pcint2
  endif
  ifneq (,$(filter atmega_pcint3,$(FEATURES_PROVIDED)))
    USEMODULE += atmega_pcint2
  endif
endif
```

But that will only work if the features are already in place when this is evaluated. But as the common Makefile is included from the CPU specific ones, this should be true.